app/javascript/components/history/HistoryComponent.js

```diff
@@ -21,16 +21,24 @@ class HistoryComponent extends React.Component {
       filteredHistories: this.props.histories,
       pageOfHistories: [],
     };
+    this.creatorFilterData = [
```

Comment:
These changes were already merged into 1.16.0, not sure why the diff is showing them here.

Reply:
I'm not seeing it in 1.16.0: https://github.com/SaraAlert/SaraAlert/blob/1.16.0/app/javascript/components/history/HistoryComponent.js#L24

Reply:
these merge commits that are on my branch are but are not in 1.16.0: https://github.com/SaraAlert/SaraAlert/pull/450/commits

Reply:
David's PR with these changes was merged into 1.16.0 so this requires further investigation then. I'll look into what's going on with 1.16.0.
https://github.com/SaraAlert/SaraAlert/pull/471/files